.travis.yml

```diff
@@ -49,14 +55,14 @@ before_script:
     # Create WordPress database.
     #- mysql -u root -e "CREATE DATABASE wordpress_test;"
     # Install CodeSniffer for WordPress Coding Standards checks.
-    - mkdir php-codesniffer && curl -L https://github.com/squizlabs/PHP_CodeSniffer/archive/3.5.0.tar.gz | tar xz --strip-components=1 -C php-codesniffer
+    - git clone -b master --depth 1 https://github.com/squizlabs/PHP_CodeSniffer.git $PHPCS_DIR
     # Install WordPress Coding Standards.
-    - mkdir wordpress-coding-standards && curl -L https://github.com/WordPress/WordPress-Coding-Standards/archive/2.1.1.tar.gz | tar xz --strip-components=1 -C wordpress-coding-standards
-    # Hop into CodeSniffer directory.
-    - cd php-codesniffer
-    # Set install path for WordPress Coding Standards
+    - git clone -b master --depth 1 https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards.git $WPCS_DIR
+    # Install PHP Compatibility sniffs.
+    - git clone -b master --depth 1 https://github.com/wimg/PHPCompatibility.git $PHPCOMPAT_DIR
```

Comment:
Hmm... why are you using an abandoned repo ? And even, why use the generic `PHPCompatibility` standard when this is a WP repo and would be better off using [`PHPCompatibilityWP`](https://github.com/PHPCompatibility/PHPCompatibilityWP) ?

Reply:
You're right should have been https://github.com/PHPCompatibility/PHPCompatibility/. As stated in the description, I tried to make the travis config work again after months of failing builds due to the things changed in the mentioned three PRs (one of them introducing PHPCompatibility). It's still not working anyway. I'll check it out again this weekend.

Reply:
Hey @jrfnl, don't really have any good input here but wanted to say hi XD

Reply:
@IanDelMar Iterating step by step towards a final solution is not a bad thing :wink: Hopefully I've given you some inspiration for the next iteration in this case.

@pattonwebz :wave:

Reply:
@IanDelMar Feel free to mention me in a next PR if you want my feedback & I'll try and look it over if I have time.

Reply:
Thank you, I’ll do that. But may I ask why you don’t make a PR?

Reply:
@IanDelMar I understand where you're coming from but:
1. I don't use and never have used Understrap.
2. I kind of maintain enough projects as it is, including PHPCompatibility and WPCS....

Reply:
> @IanDelMar Feel free to mention me in a next PR if you want my feedback & I'll try and look it over if I have time.

@jrfnl May I ask you to have a look at #1165? I'd appreciate your feedback!

Reply:
Done